### DISCUSSION TITLE
폭탄광고드랍시 세션종료

### DISCUSSION THREAD

**Discussion by generalxd** [storycraft/node-kakao#408] [Ideas]
세션이끊겨서 메시지처리를 못하는 상황이 생깁니다.

세션이야 다시 로그인하면되지만

폭탄광고가 올때마다 보고 당해야하는데..

혹시 방법이있을까요 ?

**Comment by jjungsu0822** [comment]
#363 을참고해보세요

**Reply by generalxd** [reply]
다른문제라 생각듭니다.

**Comment by ghost** [comment]
메세지가 많이 오나요? 한번에 많은 패킷을 보낼 시 연결이 끊어질 수 있습니다.

**Reply by generalxd** [reply]
> 메세지가 많이 오나요? 한번에 많은 패킷을 보낼 시 연결이 끊어질 수 있습니다.

네 좀많이오는 시간대가있는데 그때마다 끊깁니다 ㅜ

**Reply by generalxd** [reply]
> 메세지가 많이 오나요? 한번에 많은 패킷을 보낼 시 연결이 끊어질 수 있습니다.

해결방법이 없을까요 ? TPS를 제가 조절해야할까요 임의로?

**Reply by ggoek34** [reply]
해결되셧나요?
